gui-wm/gamescope: add x11-libs/libXmu to BDEPEND

g-i-g opened the pull request:
gamescope needs libXmu to build. I've confirmed in a gentoo/stage3 container that:

- the build fails without said package
- the build succeeds with said package

Bug: https://bugs.gentoo.org/912423

chewi commented:
It can't be BDEPEND because libXmu doesn't have any executable binaries, but it should be in RDEPEND + DEPEND. You can fix this here, but I'm about to bump it anyway.

g-i-g commented:
Sorry for the git mess above, but I moved libXmu to RDEPEND and squashed into one commit. ${RDEPEND} is in DEPEND, so I'm not sure if it needs to be listed twice, but please correct me if I am wrong.

chewi commented:
Now merged, thank you.
